jx_sqlite/query_table.py

```diff
@@ -229,12 +229,16 @@ def query(self, query):
                     allowNulls=allowNulls,
                     domain=domain
                 ))
+                
+            zeros = []
+            data_cubes = {}
+            for si, s in enumerate(listwrap(query.select)):
+                if s.aggregate == "count" and index_to_columns[si].push_child == ".":
+                    zeros.append(0)
+                    data_cubes[s.name] = Matrix(dims=dims, zeros=zeros[si])
```

Comment:
` and index_to_columns[si].push_child == "."` seems like an odd condition; count of anything can never be `None` , just zero, or more.  Do the tests pass with this removed?